### ISSUE TITLE
Performance regression in integration with Meijerg

### ISSUE THREAD

**Issue by moorepants** [sympy/sympy#14670]
See:

http://www.moorepants.info/misc/sympy-asv/#integrate.TimeIntegration01.time_doit_meijerg

Likely PR that introduces regression:

https://github.com/sympy/sympy/pull/13426

**Comment by asmeurer** [comment]
That benchmark looks all over the place. I don't see how a PR that only affects NumPy arrays could affect the integration code. My guess is that there is some randomness that affects the performance. To see if there really is a slowdown, you would need to lock in the PYTHONHASHSEED and possibly the random seed as well.

**Comment by moorepants** [comment]
Yes, this benchmark is problematic. If we can keep it from jumping around, that'd be great.
